src/generators/llms-txt/utils/buildApiDocLink.mjs

```diff
@@ -0,0 +1,50 @@
+import { LATEST_DOC_API_BASE_URL } from '../../../constants.mjs';
+import { transformNodeToString } from '../../../utils/unist.mjs';
+
+/**
+ * Retrieves the description of a given API doc entry. It first checks whether
+ * the entry has a llm_description property. If not, it extracts the first
+ * paragraph from the entry's content.
+ *
+ * @param {ApiDocMetadataEntry} entry
+ * @returns {string}
+ */
+const getEntryDescription = entry => {
+  if (entry.llm_description) {
+    return entry.llm_description;
+  }
+
+  const descriptionNode = entry.content.children.find(
+    child => child.type === 'paragraph'
+  );
+
+  if (!descriptionNode) {
+    return '';
+  }
+
+  return (
+    transformNodeToString(descriptionNode)
+      // Remove newlines and extra spaces
+      .replace(/[\r\n]+/g, '')
```

Comment:
Wouldn't trim be enough?

Reply:
No, because there's some line breaks in the middle of the string.